MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -585,6 +585,31 @@ public bool IsTextSubtitleStream
             }
         }
 
+        public bool IsPgsSubtitleStream
+        {
+            get
+            {
+                if (Type != MediaStreamType.Subtitle)
+                {
+                    return false;
+                }
+
+                if (string.IsNullOrEmpty(Codec) && !IsExternal)
+                {
+                    return false;
+                }
+
+                return IsPgsFormat(Codec);
+            }
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether this is a subtitle steam that is extractable by ffmpeg.
+        /// All text-based and pgs subtitles can be extracted.
+        /// </summary>
+        /// <value><c>true</c> if this is a extractable subtitle steam otherwise, <c>false</c>.</value>
+        public bool IsExtractableSubtitleStream => IsTextSubtitleStream || IsPgsSubtitleStream;
```

Comment:
I'm a bit late to the party but hopefully the celebration isn't over yet. Both `IsPgsSubtitleStream` and `IsExtractableSubtitleStream` are now part of the HTTP API. Is that intentional or were those properties only meant for the server internally?

If they are meant for the API they should be marked as nullable for backwards compatibility in the generated SDK's. (can be made not-null afterwards for 10.11).